docs/dapps/smart-contracts/using-foundry-with-the-avalanche-c-chain.md

```diff
@@ -0,0 +1,179 @@
+
+Foundry manages your dependencies, compiles your project, runs tests, deploys, and lets you interact with the chain from the command-line.
+
+## Prerequisites
+
+- You have installed [Foundry](https://github.com/foundry-rs/foundry#installation); This installation includes the `forge` and `cast` binaries used in this walk-through
+- You are familiar with [Avalanche Smart Contract Quickstart](https://github.com/ava-labs/avalanche-smart-contract-quickstart)
+- If you plan on running locally, ensure that you have installed and are familiar with [Avalanche Network Runner](../../quickstart/network-runner)
+## Getting Started
+
+This section will walk you through creating an ERC721 with Foundry and Avalanche Smart Contract Quickstart.
+Clone the Avalanche Smart Contract Quickstart repo and install its dependencies by running:
+
+```zsh
+git clone https://github.com/ava-labs/avalanche-smart-contract-quickstart.git
+cd avalanche-smart-contract-quickstart
+yarn
+```
+
+In order to deploy contracts, you need to have some AVAX. You can get testnet AVAX from the [Avalanche Faucet](https://faucet.avax.network), which is an easy way to get to play around with Avalanche. After getting comfortable with your code, you can run it on Mainnet after making the necessary changes to your workflow.
+
+## Implement a Game Item NFT
+
+We will use our example ERC721 smart contract, [`NFT.sol`](https://github.com/ava-labs/avalanche-smart-contract-quickstart/blob/3fbba0ac28f6420e9be5d2635d5f23693f80127a/contracts/NFT.sol) found in `./contracts` of our project.
+
+```ts
+//SPDX-License-Identifier: MIT
+// contracts/ERC721.sol
+
+pragma solidity >=0.6.2;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+contract NFT is ERC721 {
+  using Counters for Counters.Counter;
+  Counters.Counter private _tokenIds;
+
+  constructor() ERC721("GameItem", "ITM") {}
+
+  // commented out unused variable
+  // function awardItem(address player, string memory tokenURI)
+  function awardItem(address player)
+    public
+    returns (uint256)
+  {
+    _tokenIds.increment();
+
+    uint256 newItemId = _tokenIds.current();
+    _mint(player, newItemId);
+    // _setTokenURI(newItemId, tokenURI);
+
+    return newItemId;
+  }
+}
+```
+
+Let's examine this implementation of an NFT as a Game Item. We start by importing to contracts from our node modules. We import Openzeppelin's open source implementation of the ERC721 standard which our NFT contract will inherit from. Our constructor takes the `_name` and `_symbol` arguments for our NFT and passes them on to the constructor of the parent ERC721 implementation. Lastly we implement the `awardItem` function which allows anyone to mint an NFT to a player's wallet address. This function increments the `currentTokenId` and makes use of the `_mint` function of our parent contract.
```

Comment:
Link to OpenZeppelin's ERC721 docs here

```text
We import Openzeppelin's open source implementation of the [ERC721 standard](https://docs.openzeppelin.com/contracts/2.x/api/token/erc721)
```

Fix `Openzeppeliln's` to `OpenZeppelin's`

Reply:
will do